ftl-project.toml

```diff
@@ -1,6 +1,8 @@
 name = "ftl"
 module-dirs = ["examples/go"]
 ftl-min-version = ""
+hermit = false
+no-git = false
```

Comment:
are these required for the schema extraction changes? Similar to the `modules.test` below.

Reply:
ah not sure where these got picked up, thanks for flagging. let me check them out